private/byte.scrbl

```diff
@@ -163,3 +163,15 @@ integer between @racket[0] and @racket[255].
    (code:comment "convert byte to a vector of bits")
    (for/vector ([b (in-byte (byte 0 0 1 0 1 0 1 0))])
     b))}
+
+@defproc[(byte-hamming-weight [b byte?])
+         (integer-in 0 8)]{
+ Returns the Hamming weight of the byte. This is the same as  how many 1's are in the byte.
```

Comment:
Could you add a link to the Hamming weight wikipedia page? Use the `@hyperlink[]{}` function to add it. Since urls can be long, I usually stick it in a constant definition nearby and reference it as a variable in the `@hyperlink`, like this:

```racket
@(define foo-link "https://example.com/some/long/link")

... @hyperlink[foo-link]{displayed link text} ...
```